### PR TITLE
data:BejadiRajeshReddy18

### PR DESCRIPTION
**Pull request by BejadiRajeshReddy18**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4822"></a>

### CODE REVIEW

**Review by accodes21 — APPROVED** [review]

File: data/BejadiRajeshReddy18.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
+  "name": "RajeshReddy",
+  "displayStatsPublic":true
```
Comment (kumarsonsoff3) [inline data/BejadiRajeshReddy18.json:3]: As of latest update, we no longer need `displayStatsPublic` key. Please remove this one!

```suggestion
```

File: data/BejadiRajeshReddy18.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  "type": "personal",
+  "bio": "I'm an Devops Engineer",
+  "avatar":"https://github.com/BejadiRajeshReddy18.png"
```
Comment (kumarsonsoff3) [inline data/BejadiRajeshReddy18.json:6]: Also, we don't need `avatar` key. Remove this one from your json file too.

```suggestion
```

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]

**Review by BejadiRajeshReddy18 — COMMENTED** [review]
Linktree url isn't generated

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Thanks! 🎉

**Comment by BejadiRajeshReddy18** [comment]
> Looks good, Thanks! 🎉

Thanks
My linktree url isn't generated

**Comment by kumarsonsoff3** [comment]
> Linktree url isn't generated

It will be generated after your PR is merged & deployed. The deployment can take about 5mins after being merged.

**Pull request merged**